Classes/common/touchdb/TDRemoteRequest.m

```diff
@@ -282,58 +284,23 @@ + (BOOL)checkTrust:(SecTrustRef)trust forHost:(NSString *)host
         return NO;
     }
 }
-
-- (void)connection:(NSURLConnection *)connection didReceiveResponse:(NSURLResponse *)response
+- (void) receivedResponse:(NSURLResponse *)response
 {
+    
+    
+    //with the new filter stuff, if we hit an error we shouldn't be retying.
```

Comment:
Extra new lines, comment could be better worded (soon "filter stuff" will be old, and no one will know what "filter stuff" means).